app/feed/process_alignment_feed.py

```diff
@@ -55,12 +55,7 @@ def create_alignment_feed(
         assign_alignment_iris(
             alignment_feed,
             "solution",
-            find_alignment_solution_iris(
-                CONVERSATION_SOLUTION_NAME,
-                POPULAR_SOLUTION_NAMES,
-                POPULAR_SOLUTION_COUNT,
-                UNPOPULAR_SOLUTION_COUNT,
-            ),
+            find_alignment_solution_iris(),

```

Comment:
I guess if we pass there constants only we could omit arguments at all

Reply:
I talked about this before with Cyprian. The problem is the constants aren’t set in the app config so it can be hard to know they exist and hard to know where to find them to change them. So we kept them as inputs just to make it clearer that constants were getting used as arguments. Generally, I think it’s bad style to use scoping to keep track of access to constants. It’s better to have constants defined close to where they get used or to have them be app-wide constants. If both you and Cyprian prefer no arguments then we can just go with consensus instead of my concerns about style and scoping.

Reply:
well, since it's not personalized at all and will be improved in the future (I hope so, otherwise it doesn't make sense) I would keep it as is for now.
also, I suggest to rename it to avoid confusion. 

I thought this part of the project is crucial to have to be able to provide meaningful conversations. when we are planning to adjust this?

Reply:
> Generally, I think it’s bad style to use scoping to keep track of access to constants. It’s better to have constants defined close to where they get used or to have them be app-wide constants.

I think what happened here is that I assumed you weren't initially talking about app-wide constants, as I didn't find similar precedents in the codebase, and I assumed you weren't talking about constants declared inside functions, as then I don't think of them as "constants", which left me with this awkward middle ground. :) But indeed, we can improve this at another time.